shaders/basic/frag.glsl

```diff
@@ -40,12 +39,9 @@ void main()
 	vec3 light_dir = normalize(-dir_light.direction);
     vec3 light_intensity = dir_light.color;
 	vec3 result;
-	if (texture_enabled){		
+	if (texture_enabled){
 		vec3 tex_color;
-		if(texture_noise)
-			tex_color = vec3(texture(texture_map, frag_tex_coord + noise));
-		else
-			tex_color = vec3(texture(texture_map, frag_tex_coord));
+        tex_color = vec3(texture(texture_map, frag_tex_coord));
```

Comment:
Please be consistent about tabs vs. spaces (not just here, throughout the commit)